### DISCUSSION TITLE
Repositório excluído

### DISCUSSION THREAD

**Discussion by ws-adm** [community/community#86600] [Accessibility]
### Select Topic Area

Question

### Body

Bom dia a todos, sabem se é possível recuperar um repositório excluído? 
vi no gitHUB que é possível tem a opção, mas no gitLab não vi.

**Comment by ghostinhershell** [comment]
Hi @ws-adm , thanks for participating! I do want to note a couple of things here:

1. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them. I've gone ahead and moved this to the correct category for you.

2. We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.

3. With GitHub, a deleted repository can be restored within 90 days, unless the repository was part of a fork network that is not currently empty. You can learn more about that here: [Restoring a deleted repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository). I can't speak to GitLab's recovery process, however, you will need to reach out to their support to get answer on that.
